src/base/_defaults.scss

```diff
@@ -48,17 +48,13 @@ ul {
 }
 
 /**
- * 1. Forces the viewport to be filled. Necessary for the footer to
- *    "stick" to the bottom of short pages.
- *    @see https://css-tricks.com/couple-takes-sticky-footer/#there-is-grid
```

Comment:
The `block-size: 100%` code already existed in the `.o-page` class and I changed it to `100vh` in the `.o-page` class so the 100% wouldn't need to be propagated all the way to `html` here.

This fixes the double-scrollbar issue, but I'm not completely sure why it had the issue as-is.